apps/test/unit/templates/MultipleSectionAssignerTest.js

```diff
@@ -151,29 +150,61 @@ describe('MultipleSectionsAssigner', () => {
       courseVersionId: assignedCourseANDUnitSection.courseVersionId,
     });
 
-    // Check that courses 1-6 have TeacherOptions and course 7 does not.
-    for (let i = 0; i < wrapper.instance().props.sections.length; i++) {
-      wrapper.instance().props.sections[i].participantType ===
-      wrapper.instance().props.participantAudience
-        ? expect(
-            wrapper
-              .find('TeacherSectionOption')
-              .filterWhere(
-                n =>
-                  n.props().section.id ===
-                  wrapper.instance().props.sections[i].id
-              )
-          ).to.exist
-        : expect(
-            wrapper
-              .find('TeacherSectionOption')
-              .filterWhere(
-                n =>
-                  n.props().section.id ===
-                  wrapper.instance().props.sections[i].id
-              )
-          ).to.have.lengthOf(0);
-    }
+    const assignableSections = fakeTeacherSectionsForDropdown.filter(
```

Comment:
We can't access the props of the component like we used to; we only have access to the props placed on the root node. See https://enzymejs.github.io/enzyme/docs/api/ShallowWrapper/props.html for more about this. As such, I couldn't access all the sections via the `sections` prop.